services/worker-manager/src/providers/aws.js

```diff
@@ -108,14 +105,15 @@ class AwsProvider extends Provider {
         });
       }
 
+      let spawned;
       try {
         spawned = await this.ec2s[config.region].runInstances({
           ...config.launchConfig,
 
           UserData: userData.toString('base64'), // The string needs to be base64-encoded. See the docs above
 
-          MaxCount: Math.min(toSpawnCounter, toSpawnPerConfig),
-          MinCount: Math.min(toSpawnCounter, toSpawnPerConfig),
+          MaxCount: Math.min(toSpawnCounter, Math.ceil(toSpawnPerConfig / config.capacityPerInstance)),
```

Comment:
I think we need to divide `toSpawnCounter` as well, because it counts _capacity_, but we are sending the amount of _instances_ we need to `runInstances`...